libcxx/docs/ReleaseNotes/18.rst

```diff
@@ -133,6 +133,13 @@ ABI Affecting Changes
   results in an ABI break, however in practice we expect uses of ``std::projected`` in ABI-sensitive places to be
   extremely rare. Any error resulting from this change should result in a link-time error.
 
+- Under the unstable ABI, the internal alignment requirements for heap allocations
+  inside std::string has decreased from 16 to 8 This save memory since string requests fewer additional
```

Comment:
s/std::string/``std::string``/ so it will be rendered properly, the same for other places.